wdtk-dumpfiles/src/main/java/org/wikidata/wdtk/dumpfiles/WikibaseRevisionProcessor.java

```diff
@@ -154,6 +164,10 @@ public void processPropertyRevision(MwRevision mwRevision) {
 
 	}
 
+	private boolean isWikibaseRedirection(MwRevision mwRevision) {
+		return redirectionsPattern.matcher(mwRevision.getText()).matches();
```

Comment:
Done with `.contains` and string `"redirect":` in order to don't detect other structures starting with `{"entity":` wrongly as redirects if they are introduced in the future.